packages/sku/src/services/vite/helpers/server/createViteServer.ts

```diff
@@ -22,7 +22,7 @@ export const createViteServer = async (skuContext: SkuContext) =>
           target: 'node',
           type: 'static',
         }),
-        skuContext.httpsDevServer && skuViteHttpsDevServer(skuContext),
+        getHttpsDevServerPlugin(skuContext),
```

Comment:
Is the name change needed? Would be nice to keep it the same as the others.

```suggestion
        skuViteHttpsDevServer(skuContext),
```

Reply:
I feel like the original name wasn't really correct though. It never returned a server, it was really a getter for some plugins. It's still that, so I figured I'd name it as such. Honestly the `skuVite` prefix is redundant, and I want to rename the others as well. Happy to undo the rename in this PR and rename everything separately though.